src/main/actions.js

```diff
@@ -11,6 +12,9 @@ export function setupActions(store) {
   // Clear update notice
   store.dispatch(updateInstalled());
 
+  // Update the menu
+  store.subscribe(debounce(() => setupMenu(store), 500));
```

Comment:
Why debounce menu creation? Isn't 500ms maybe a bit much?

Reply:
Its not menu creation. Everytime the store updates (any action regardless of it having anything to do with menu or not) this re-draws the menu which is slightly heavy process. And sometimes this function (subcribed function) gets called 20 times in 100ms. So just wanted to avoid performance issues and lags

Reply:
👍